tests/manual/forwardrenderer_scene/scenecontroller.cpp

```diff
@@ -44,6 +44,10 @@ SceneController::SceneController(QObject *parent)
     , m_threshold(.5)
     , m_opacityMaskEffect(false)
     , m_opacityMaskPremultipliedAlpha(false)
+    , m_depthOfFieldEffect(false)
+    , m_depthOfFieldRadius(16.f)
+    , m_depthOfFieldRange(3.f)
+    , m_depthOfFieldDistance(5.f)
```

Comment:
can we change this to be 10 or something? As it it, in the demo, when you turn on DOF, everything goes blurry instead of one object remaining sharp while the rest going blurry. You have to move the distance to make the effect noticeably different from just blurring.